docs/manuals/core/entities/scheduling-and-job-execution.md

```diff
@@ -38,7 +38,9 @@ task = scenario.predicting
 tp.submit(task, wait=True, timeout=3)
 ```
 
-_timeout_ can be an integer or a float. By default, _wait_ is False and _timeout_ is None. If _wait_ is True and _timeout_ is not specified or None, there is no limit to the wait time. If _wait_ is True and _timeout_ is specified, taipy will wait until all the submitted jobs are finished or up to _timeout_ seconds.
+_timeout_ can be an integer or a float. By default, _wait_ is False and _timeout_ is None. If _wait_ is True and
```

Comment:
All this information should be in the RefMan, and there only unless you stress on a specific parameter.
And I suspect there's a cross-ref not too far away anyway...